pkg/i18n/japanese.go

```diff
@@ -20,87 +20,88 @@ Thanks for using lazygit! Seriously you rock. Three things to share with you:
-		// LcRebaseBranch:                      "rebase checked-out branch onto this branch",
+		NotEnoughSpace:            "パネルの描画に十分な表示領域がありません",
+		DiffTitle:                 "差分",
+		FilesTitle:                "ファイル",
+		BranchesTitle:             "ブランチ",
+		CommitsTitle:              "コミット",
+		StashTitle:                "Stash",
+		UnstagedChanges:           `ステージされていない変更`,
+		StagedChanges:             `ステージされた変更`,
+		MainTitle:                 "メイン",
+		MergeConfirmTitle:         "マージ",
+		StagingTitle:              "メインパネル (Staging)",
+		MergingTitle:              "メインパネル (Merging)",
+		NormalTitle:               "メインパネル (Normal)",
+		LogTitle:                  "ログ",
+		CommitMessage:             "コミットメッセージ",
+		CredentialsUsername:       "ユーザ名",
+		CredentialsPassword:       "パスワード",
+		CredentialsPassphrase:     "SSH鍵のパスフレーズを入力",
+		PassUnameWrong:            "パスワード, パスフレーズまたはユーザ名が間違っています。",
+		CommitChanges:             "変更をコミット",
+		AmendLastCommit:           "最新のコミットにamend",
+		AmendLastCommitTitle:      "最新のコミットにamend",
+		SureToAmend:               "最新のコミットに変更をamendします。よろしいですか? コミットメッセージはコミットパネルから変更できます。",
+		NoCommitToAmend:           "amend可能なコミットが存在しません。",
+		CommitChangesWithEditor:   "gitエディタを使用して変更をコミット",
+		StatusTitle:               "ステータス",
+		LcNavigate:                "移動",
+		LcMenu:                    "メニュー",
+		LcExecute:                 "実行",
+		LcToggleStaged:            "ステージ/アンステージ",
+		LcToggleStagedAll:         "すべての変更をステージ/アンステージ",
+		LcToggleTreeView:          "ファイルツリーの表示を切り替え",
+		LcOpenMergeTool:           "git mergetoolを開く",
+		LcRefresh:                 "リフレッシュ",
+		LcPush:                    "push",
+		LcPull:                    "pull",
+		LcScroll:                  "スクロール",
+		MergeConflictsTitle:       "マージコンフリクト",
+		LcCheckout:                "チェックアウト",
+		LcFileFilter:              "ファイルをフィルタ (ステージ/アンステージ)",
+		FilterStagedFiles:         "ステージされたファイルのみを表示",
+		FilterUnstagedFiles:       "ステージされていないファイルのみを表示",
+		ResetCommitFilterState:    "フィルタをリセット",
+		NoChangedFiles:            "変更されたファイルがありません",
+		NoFilesDisplay:            "表示するファイルがありません",
+		NotAFile:                  "ファイルではありません",
+		PullWait:                  "Pull中...",
+		PushWait:                  "Push中...",
+		FetchWait:                 "Fetch中...",
+		LcSoftReset:               "softリセット",
+		AlreadyCheckedOutBranch:   "ブランチはすでにチェックアウトされています。",
+		SureForceCheckout:         "全てのローカルな変更が失なわれます。本当に強制的にチェックアウトしますか?",
+		ForceCheckoutBranch:       "ブランチを強制的にチェックアウト",
+		BranchName:                "ブランチ名",
+		NewBranchNameBranchOff:    "新規ブランチ名 ('{{.branchName}}' に作成)",
+		CantDeleteCheckOutBranch:  "チェックアウト中のブランチは削除できません!",
+		DeleteBranch:              "ブランチを削除",
+		DeleteBranchMessage:       "ブランチ '{{.selectedBranchName}}' を削除します。よろしいですか?",
+		ForceDeleteBranchMessage:  "'{{.selectedBranchName}}' はマージされていません。本当に削除しますか?",
+		LcRebaseBranch:            "チェックアウトされたブランチをこのブランチにリベース",
```

Comment:
```suggestion
		LcRebaseBranch:            "チェックアウトされたブランチをこのブランチにrebase",
```

`rebase` should not be translated for the UX when searching for menu items.